app/views/pages/about.html.erb

```diff
@@ -81,7 +81,7 @@
       </div>
       <div class="about-row">
         <div class="about-key">
-          <img src="<%= cloudinary "https://thepracticaldev.s3.amazonaws.com/i/pm1d5npekd2j6ts2hcte.jpeg" %>" width="250" alt="Peter Frank" />
+          <%= cl_image_tag("peter-frank.jpg", width: 250, crop: :scale, fetch_format: "auto", flags: ["progressive"], quality: "auto", type: :asset, alt: "Peter Frank") %>
```

Comment:
As @rhymes mentioned, the about page in prod is different, so we actually don' need this to be a `cl_image_tag`. Same for Peter's photo below.

Reply:
@nickytonline Changed to normal `image_tag`.